tiny-postgresql-event-store/src/main/kotlin/jp/veka/factory/PostgresConnectionFactoryImpl.kt

```diff
@@ -0,0 +1,15 @@
+package jp.veka.factory
+
+import java.sql.Connection
+import java.sql.DriverManager
+
+class PostgresConnectionFactoryImpl(databaseUrl: String, user: String, password: String) : PostgresConnectionFactory {
```

Comment:
I don't think we have to write such kind of connection-managing code. There should be a plenty of existing solutions including connection-pools (like [Hikary](https://github.com/brettwooldridge/HikariCP)).